lambda-functions/Lambda/psycopg2-no-SQLAlchemy/access_psycopg2.py

```diff
@@ -0,0 +1,37 @@
+import psycopg2
+from subprocess import call
+
+# as the name implies, access the RDS database with pyscopg2 without the need of an ORM like SQLAlchemy
+# DBInstanceIdentifier can be found by running the command after the ` in your shell
+# this is a dependency file of our lambda function postgres_makeconn.py
+
+db_host = call("aws rds describe-db-instances | jq -r '.DBInstances[]|select(.DBInstanceIdentifier=='rds-reddit-data').Endpoint|.Address'", shell=True)
```

Comment:
Consider making this a shell script in the `scripts` directory at the root of the repo